README.md

```diff
@@ -182,14 +182,17 @@ docker pull eclipse-mosquitto:latest
 ```
 2.  `BROKER_ENDPOINT` defined in `demos/mqtt/mqtt_demo_basic_tls/demo_config.h` can now be set to `localhost`.
 
-3. For TLS communication with Mosquitto broker, server and CA credentials need to be created. Use OpenSSL commands to generate the credentials for the Mosquitto server.
+3. For TLS communication with Mosquitto broker, server and CA credentials need to be created. Use OpenSSL commands to generate the credentials for the Mosquitto server.  
+**Note:** Make sure to use different Subject details between the Root CA and server certificates; otherwise, SSL handshake fails with exactly same Subject details in both the certificates.
```

Comment:
Sorry I didn;t see you make this change. Only the Common Name needs to be different. ALl other fields can be the same.